madagascar-conservation-and-development.csl

```diff
@@ -0,0 +1,338 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" et-al-min="4" et-al-use-first="1" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
since you do set et-al for both citation and bibliography, it has no use here (and will confuse people working with the style in the future. Please delete ```et-al-min="4" et-al-use-first="1"```